nestkernel/connector_base.h

```diff
@@ -262,12 +262,10 @@ class Connector : public ConnectorBase
     C_[ lcid ].set_status( dict, static_cast< GenericConnectorModel< ConnectionT >& >( cm ) );
   }
 
-
-  Connector< ConnectionT >&
+  void
```

Comment:
these changes are logically distinct from the changes that this PR is implementing, aren't they? if so, they should be moved to a separate PR